### PR TITLE
Update Security Guide/PCI-DSS Guide to sentence case (WIP)

### PR DESCRIPTION
**Pull request by sknorr**
### Description

* The new capitalization rules are documented at https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-capitalization, read carefully
* This is the result of an automatic conversion and will most likely need further adjustments
  * Make sure to review the entire diff
  * Entity files have not been changed and need to be updated manually
  * Some product/project names may have been inadvertently changed to the wrong capitalization
  * Some titles may have been missed, either because they're split across multiple source lines or because of improper markup
* Update the conversion commit and remove the (WIP) from the commit message before merging
* This is the fourth PR in a series of 14 PRs

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ---------- 
| [x] 15 SP3  | (`master` only)   
| [ ] 15 SP2  | [ ] 
| [ ] 15 SP1  | [ ] 
| [ ] 15 SP0  | [ ] 

| Code 12     | Backport Done
| ----------  | ---------- 
| [ ] 12 SP5  | [ ] 
| [ ] 12 SP4  | [ ] 
| [ ] 12 SP3  | [ ] 
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/common_criteria.xml [hunk 1]
```diff
@@ -188,7 +188,7 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Service Isolation&mdash;Run Different Services on Separate
+     <term>Service isolation&mdash;run different services on separate
      Systems</term>
```
Comment (lvicoun) [inline xml/common_criteria.xml:192]: Here it spans to another line and the Systems is still capitalized.

File: xml/security_ad_support.xml [hunk 2]
```diff
@@ -741,7 +741,7 @@
        <itemizedlist>
         <listitem>
          <formalpara>
-          <title>If the Local Host Name Does Not Match the Host Name Set on the
+          <title>If the local host name does not match the host name set on the
```
Comment (lvicoun) [inline xml/security_ad_support.xml:744]: Here again, text spans line and changes were not applied.

File: xml/common_criteria.xml (hunk 1 above)
Comment (sknorr) [reply xml/common_criteria.xml:192]: Please fix this manually, then. This is something I cannot easily update within the script.

**Review by lvicoun — CHANGES_REQUESTED** [review]
There are some issues with line spanning text and omitting a variablistentry from changes.

**Review by lvicoun — APPROVED** [review]

**Review by lvicoun — APPROVED** [review]

**Pull request closed without merging**